storage/azblob/azblob.go

```diff
@@ -450,9 +467,7 @@ func (c *client) DeleteRequest(
 			Reason:  err,
 		}
 	}
-	now := time.Now().UTC()
-	exp := now.Add(duration)
-	uri, err := bc.GetSASURL(sas.BlobPermissions{Delete: true}, now, exp)
```

Comment:
in both the delete and put requests we effectively removed that call, just to make me calm: why?